admin/acceptance.py

```diff
@@ -878,7 +879,7 @@ def _libcloud_runner(self, package_source, dataset_backend,
             dataset_backend=dataset_backend,
             dataset_backend_configuration=self.dataset_backend_configuration(),
             variants=self['variants'],
-            num_nodes=int(os.environ.get("FLOCKER_ACCEPTANCE_NUM_NODES", "2")),
+            num_nodes=self['number-of-nodes'],
```

Comment:
Argh.  While this is clearly an improvement, this means now we have to make sure there are no users of the environment variable that are going to break because of this change.